types/test.ts

```diff
@@ -78,6 +78,34 @@ const transaction = new StellarSdk.TransactionBuilder(account, {
         preAuthTx: "da0d57da7d4850e7fc10d2a9d0ebc731f7afb40574c03395b17d49149b91f5be"
       }
     })
+  ).addOperation(
```

Comment:
This looks great, I didn't notice that there are corresponding tests for the types, which caused me to miss them.

Reply:
Yep! Likewise.